src/intrinsic/mod.rs

```diff
@@ -385,7 +386,8 @@ impl<'a, 'gcc, 'tcx> IntrinsicCallMethods<'tcx> for Builder<'a, 'gcc, 'tcx> {
         };
 
         if !fn_abi.ret.is_ignore() {
-            if let PassMode::Cast { cast: ty, .. } = &fn_abi.ret.mode {
+            // The reference was changed to clone to comply to clippy.
```

Comment:
Please remove this comment.

Reply:
Done in tempdragon@3b01fba0f5ecfd7e929fcf1948945ae75479c38d